# Unformatted Error Output:
# File "tests/__tests__/refmterr/__fixtures__/type_UnboundValue/type_UnboundValue_2.ml", line 1, characters 21-24:
# Error: Unbound value nvm


 ERROR  tests/__tests__/refmterr/__fixtures__/type_UnboundValue/type_UnboundValue_2.ml:1 21-24 

1 â”† let callMe perhaps = nvm 1

The value named nvm can't be found. Could it be a typo?


